### ISSUE TITLE
Memory usage

### ISSUE THREAD

**Issue by FlorianLudwig** [wagtail/bakerydemo#238]
Is it expected behavior that the app container needs 8gb of ram?

**Comment by thibaudcolas** [comment]
Hey @FlorianLudwig 👋 could you explain what container you’re referring to and how you’re measuring this?

For me locally `docker stats` reports the app container at `155MiB`, and I know that outside of Docker the bakerydemo easily runs in Heroku dynos capped at 500MB.

**Comment by FlorianLudwig** [comment]
Hi @thibaudcolas,

I am referring to the container created by docker-compose.

what I did was, following the readme:
```
git clone https://github.com/wagtail/bakerydemo.git
cd bakerydemo
docker-compose up --build -d
docker-compose run app /venv/bin/python manage.py load_initial_data
docker-compose up
```

It wasn't that straight, i had to fiddle a little:

 * the first build didn't work, I had https://github.com/wagtail/bakerydemo/pull/235 applied
 * starting elastic search failed because `vm.max_map_count` was to low.

So the order in which the containers started and restarted might be of influence to reproduce this.

But... I can't reproduce it anymore.  I checked out current master and now the memory usage is at 50MB.

Curious: It didn't leak up to that memory it allocated it in one go as also run into an error message that states it failed to allocate about 8gb memory (which I didn't have free at that time).

Anyway, as I am unable to reproduce this I am closing the issue.
